consensus/consensus.go

```diff
@@ -340,7 +340,7 @@ func accumulateRewards(
 	if err != nil {
 		return ctxerror.New("cannot create group sig mask").WithCause(err)
 	}
-	if err := mask.SetMask(parentHeader.CommitBitmap); err != nil {
+	if err := mask.SetMask(header.LastCommitBitmap); err != nil {
```

Comment:
Add a todo to take care of epoch transition. Because right it's matching current bitmap with committee of last block, which only works if there is committee change.

Reply:
added todo